libs/patchwheel/filter.go

```diff
@@ -0,0 +1,57 @@
+package patchwheel
```

Comment:
Feels like this belongs in `libs/python` more than patchwheel. WDYT?

Reply:
libs/python is about finding python interpreter.

libs/patchwheel is about patching wheel but also parsing wheel names. It's a better fit but perhaps the name should be changed to something like wheel.